packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -114,7 +113,7 @@ export class AddJob {
       })
     );
 
-    const delay = digestAmount ?? delayAmount;
+    const delay = command.filtered ? 0 : digestAmount ?? delayAmount;
```

Comment:
If delay or digest step is filtered we should not delay the step but execute it directly.